### PR TITLE
Merge uBO Annoyances into the Annoyances

### PR DESCRIPTION
**Pull request by ryanbr**
Easier to just merge the Annoyances related lists into one, so we get less confusion and get more benefits.

Since uBO Annoyances should be used with other lists, rather than standalone

### CODE REVIEW

File: filter_lists/list_catalog.json [hunk 1]
```diff
@@ -206,7 +206,7 @@
     },
     {
         "uuid": "67E792D4-AE03-4D1A-9EDE-80E01C81F9B8",
-        "title": "Fanboy's Annoyances",
+        "title": "Fanboy's Annoyances (and combined with uBO Annoyances list)",
```
Comment (antonok-edm) [inline filter_lists/list_catalog.json:209]: Maybe `Fanboy's Annoyances + uBO Annoyances`... or just `Annoyances`?

@rmcfadden3 any opinions here?

Comment (ryanbr) [reply filter_lists/list_catalog.json:209]: `Fanboy's Annoyances + uBO Annoyances` works

Comment (rmcfadden3) [reply filter_lists/list_catalog.json:209]: Yeah, agreed: `Fanboy's Annoyances + uBO Annoyances` works just fine.

The only question I might have is whether people know what `uBO` stands for. If there's any concern around that, you could expand to `Fanboy's Annoyances + uBlock Annoyances`. But honestly either option should work.

Comment (antonok-edm) [reply filter_lists/list_catalog.json:209]: Good point - `Fanboy's Annoyances + uBlock Annoyances` sounds good. The original entry was `uBlock Annoyances` anyways.

Comment (ryanbr) [reply filter_lists/list_catalog.json]: Adjusted

**Review by antonok-edm — APPROVED** [review]

**Review by antonok-edm — APPROVED** [review]

**Pull request merged**